doc/modules/feature_extraction.rst

```diff
@@ -916,6 +916,30 @@ Some tips and tricks:
 
     (Note that this will not filter out punctuation.)
 
+
+    The following example will, for instance, transform British spelling to American
+    spelling::
+
+        >>> import re
+        >>> def to_british(tokens):
+        ...     for t in tokens:
+        ...         t = re.sub(r"(...)our$", r"\1or", t)
+        ...         t = re.sub(r"([bt])re$", r"\1er", t)
+        ...         t = re.sub(r"([iy])s(e$|ing|ation)", r"\1z\2", t)
+        ...         t = re.sub(r"ogue$", "og", t)
+        ...         yield t
+        ...
+        >>> class CustomVectorizer(CountVectorizer):
+        ...     def build_tokenizer(self):
+        ...         base = super(CustomVectorizer, self).build_tokenizer()
+        ...         return lambda doc: list(to_british(base(doc)))
+        ...
+        >>> print(CustomVectorizer().build_analyzer()(u"color colour")) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+        [...'color', ...'color']
+
+    The same approach could be used for stemming.
```

Comment:
The number normalization idea is also performed [in this example](https://github.com/scikit-learn/scikit-learn/blob/master/examples/bicluster/plot_bicluster_newsgroups.py) in our codebase. How about modifying that example to use the same inheritance trick (to avoid duplication of the tokenizing functionality), and then reference it from here?